features/user_can_sign_in.feature

```diff
@@ -0,0 +1,33 @@
+Feature: User can sign in
+  As a user
+  In order to access my account
+  I should be able to log in
+
+  Background:
+    Given following user exist
+      | email             | password    |
+      | harald@norge.no   | OsloOslo123 |
+
+  Scenario: User successfully logs in
+    Given I visit the site
+    And I click "Sign in"
```

Comment:
These steps could be moved to the background

Reply:
Fixed